lib/data_integrity_utils.pm

```diff
@@ -42,8 +51,12 @@ sub get_image_digest {
 }
 
 =head2 verify_checksum
-Verify image data integrity by comparing SHA256 checksums
+
+ verify_checksum();
+
+Verify image data integrity by comparing SHA256 checksums.
 Returns error message in case of failure, empty string in case of success
+
 =cut
 sub verify_checksum {
     my ($dir_path) = shift;
```

Comment:
Same here: There is a parameter here, this needs to be mentioned in the documentation.